packages/driver-interface/src/services/google.ts

```diff
@@ -35,13 +35,21 @@ export const getAddressFromCoordinate = (coordinate: {
       console.log('Error getting google geocode data: ', err.message)
     )
 
-export const getDirectionsFromActivities = (
-  activities: Instruction[]
-): string =>
-  activities.reduce(
-    (result, { address }) => result.concat(`/${address.lat},${address.lon}`),
-    'https://www.google.com/maps/dir'
-  )
+export const getDirectionsFromInstructions = (
+  instructionGroups: Instruction[][]
+) =>
+  instructionGroups
+    .flatMap((instructionGroup: Instruction[]) =>
+      instructionGroup.map(({ address }) => address)
+    )
+    .filter(
```

Comment:
Basically the same thing is done in convertInstructionsToSummaryList, maybe they could share base functionality, as Instruction[] are grouped by address.

Reply:
Don't see what you mean here.